contrib/cryptomb/private_key_providers/source/cryptomb_private_key_provider.cc

```diff
@@ -29,6 +29,54 @@ void CryptoMbContext::scheduleCallback(enum RequestStatus status) {
   schedulable_->scheduleCallbackNextIteration();
 }
 
+bool CryptoMbEcdsaContext::ecdsaInit(const uint8_t* in, size_t in_len) {
+  if (ec_key_ == nullptr) {
+    return false;
+  }
+
+  const EC_GROUP* group = EC_KEY_get0_group(ec_key_.get());
+  priv_key_ = EC_KEY_get0_private_key(ec_key_.get());
+  if (group == nullptr || priv_key_ == nullptr) {
+    return false;
+  }
+
+  const BIGNUM* order = EC_GROUP_get0_order(group);
+  if (order == nullptr) {
+    return false;
+  }
+
+  // Create an ephemeral key.
+  ctx_ = bssl::UniquePtr<BN_CTX>(BN_CTX_new());
+  if (ctx_ == nullptr) {
+    return false;
+  }
+  BN_CTX_start(ctx_.get());
+  k_ = BN_CTX_get(ctx_.get());
```

Comment:
the doc say this could return nullptr, so we probably better to check that.

```
// BN_CTX_get returns a new |BIGNUM|, or NULL on allocation failure. Once
// |BN_CTX_get| has returned NULL, all future calls will also return NULL until
// |BN_CTX_end| is called.
OPENSSL_EXPORT BIGNUM *BN_CTX_get(BN_CTX *ctx);
```

Reply:
Added null check, thanks.